packages/web/src/pages/rewards-page/Tiers.tsx

```diff
@@ -34,8 +34,9 @@ const { pressDiscord } = vipDiscordModalActions
 const { getUserId } = accountSelectors
 
 const messages = {
-  title: 'Reward Perks',
-  subtitle: 'Keep $AUDIO in your wallet to enjoy perks and exclusive features.',
+  title: '$AUDIO VIP TIERS',
+  subtitle:
+    'Hold $AUDIO in your wallet to unlock VIP Tiers and earn access to exclusive features and more!',
```

Comment:
feel like this copy could be a bit better..think the double and feels off a bit? 

`"Holding $AUDIO in your wallet grants you access to VIP Tiers, exclusive features, and more!"` maybe?

Reply:
copy from figma, also not new

Reply:
hmm ok, will ask design about it and update in the future based on the answer